ansible/templates/minigraph_png.j2

```diff
@@ -76,8 +76,8 @@
 {% endfor %}
 {% endfor %}
 {% endif %}
+{% if switch_type is not defined or (switch_type != 'voq' and switch_type != 'chassis-packet') %}
```

Comment:
`DeviceInterfaceLink` is also required for multi asic linecards,  right?

Reply:
Right, I agree. We can remove this check completely so that for all multi-asic scenario, we see the DeviceInterfaceLink updated correctly. 
There will be no change in the supervisor minigraph or config_db.
There will be no change in minigraph or config_db for single asic linecard as well.
There will be no change in namespace config (config_dbX.json) for multi-asic linecard.
The onlu change I see in multi-asic linecard is that, DEVICE_NEIGHBOR table in default config_db gets populated.
Sample change:
```
    "DEVICE_NEIGHBOR": {
        "Ethernet1/5": {
            "name": "ASIC1",
            "port": "Eth0-ASIC1"
        },
        "Ethernet1/1": {
            "name": "ASIC0",
            "port": "Eth0-ASIC0"
        },
        "Ethernet1/2": {
            "name": "ASIC0",
            "port": "Eth1-ASIC0"
        },
        "Ethernet1/6": {
            "name": "ASIC1",
            "port": "Eth1-ASIC1"
        },
```
In VoQ chassis multi-asic Linecard, in config_db.json for default namespace, I see that DEVICE_NEIGHBOR table gets populated, and PORT table gets populated with all ports and not just the ports with external neighbors.
These changes should not cause any functionality impact.

Reply:
@SuvarnaMeenakshi  - Sorry for not testing this out earlier.

 With your changes incorporated, the generated minigraph on a multi-asic linecard does not have the DeviceInterfaceLink with ChassisInternal as true. As an example, with your changes we are missing

```
<DeviceLinkBase i:type="DeviceInterfaceLink">
  <ElementType>DeviceInterfaceLink</ElementType>
  <Bandwidth>400000</Bandwidth>
  <ChassisInternal>true</ChassisInternal>
  <EndDevice>ASIC0</EndDevice>
  <EndPort>Eth2-ASIC0</EndPort>
  <FlowControl>true</FlowControl>
  <StartDevice>ixre-egl-board4</StartDevice>
  <StartPort>Ethernet2</StartPort>
  <Validate>true</Validate>
</DeviceLinkBase>
```

Reply:
Thank you, so we could go ahead and remove this check 
{% if inventory_hostname not in device_conn or port_alias[loop.index - 1] in device_conn[inventory_hostname] %} or the new check that I added {% if switch_type is not defined or (switch_type != 'voq' and switch_type != 'chassis-packet') %} 
completely from the template so we generate the internal links.

Reply:
I agree, we can generate the `DeviceInterfaceLink` for all platforms.
@sanmalho-git , can you confirm.

Reply:
Make sense

Reply:
Updated diff as discussed.